test/node/buffer/arraybuffer.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
```

Comment:
are all of these files 1:1 copied from node.js?

what is our plan for syncing this? should it even be pasted and committed into our repo 1:1, this is a lot of text content that is (hopefully) automatically sourced.

Reply:
It's 1:1, aside from prettier formatting. I thought it would be useful to have them checked out, but we could just have them to gitignore and have the script clone them over.